app/services/importers/tulsa_blotter/daily_import.rb

```diff
@@ -0,0 +1,116 @@
+module Importers
+  module TulsaBlotter
+    class DailyImport < ApplicationService
+      def perform
+        save_pages
+      end
+
+      def save_pages
+        page_number = 1
+        incarcerated_ids = []
+        while (response = page_json(page_number))
+          json = JSON.parse(response)
+          save_page(json)
+          page_number += 1
+          incarcerated_ids += (json['Data'].map { |val| val['Booking ID'] })
+        end
+        track_released(incarcerated_ids)
```

Comment:
send the full list of the inmates currently incarcerated so we can flag the ones not in there as released